hashicorp/dev-portal
# Install page enhancement: Move OS downloads from tabs to individual cards

## 🔗 Relevant links



- [Preview link](https://dev-portal-git-heat-featinstallos-cards-hashicorp.vercel.app/vault/install) 🔎
- [Asana task](https://app.asana.com/0/1204817897905865/1205911880806862/f) 🎟️
- [Figma](https://www.figma.com/file/308QXXjFalJSFaCvmFU50L/Install%2FDownload-Template?type=design&node-id=926-11525&mode=design&t=dxobrdCfnqNtkqsw-0)

## 🗒️ What

- Move OS download content from tabs to individual cards per Figma mocks
- Added a `'middle'` option for the merchandising slot.
  - This allows for Boundary Desktop Client to be passed as the `merchandisingSlot` in `/pages/boundary/install`

## 📸 Design Screenshots

<img width="500" alt="Screenshot 2023-11-20 at 2 34 37 PM" src="https://github.com/hashicorp/dev-portal/assets/55773810/28e47c78-dc0c-49fa-ba92-fe1dcb369052">

### Boundary Desktop client
<img width="568" alt="Screenshot 2023-11-21 at 4 50 10 PM" src="https://github.com/hashicorp/dev-portal/assets/55773810/7339e4f9-716b-437f-a7c5-c5a2e2d4ec3e">

## 🧪 Testing

- Navigate to current [Vault install page](https://developer.hashicorp.com/vault/install) and see the changes
- Navigate to the [Vault install preview page](https://dev-portal-git-heat-featinstallos-cards-hashicorp.vercel.app/vault/install) to see the changes
- Navigate to [Boundary install page](https://dev-portal-git-heat-featinstallos-cards-hashicorp.vercel.app/boundary/install) to see the Desktop client

## 💭 Anything else?

- Making each OS download card heading an anchor link ([example here](https://developer.hashicorp.com/well-architected-framework/security/security-sensitive-data)) will be addressed in a future PR once the page restructuring is complete.
